public/main.js

```diff
@@ -0,0 +1,170 @@
+//Импорт классов, почему то, сделать не вышло, так что они все здесь
+const search_btn = document.getElementById('search_btn');
+const parentBlock = document.getElementById('music_list');
+const libraryStorage = document.getElementById('libraryStorage');
+let state = '';
+
+//Запрос на сервер
```

Comment:
дока пишется не так

должно быть
```suggestion
/**
 *Запрос на сервер
 */
```